string command
---
int32 result
